assets/app/views/project.html

```diff
@@ -54,29 +56,27 @@ <h2 class="service" ng-init="numPorts = service.spec.ports.length">{{serviceId}}
                   exists="deploymentConfigs[deploymentConfigId]"
                   different-service="!deploymentConfigsByService[serviceId][deploymentConfigId]">
                 </deployment-config-metadata>
-                <div ng-repeat="deployment in deployments | orderObjectsByDate : true" style="margin-top: 10px; text-align: center;">
+                <div ng-repeat="deployment in deployments | orderObjectsByDate : true track by (deployment | uid)" style="margin-top: 10px; text-align: center;" class="osc-object" kind="ReplicationController" resource="deployment" ng-if="(podsByDeployment[deployment.metadata.name] | hashSize) > 0">
                   <!-- TODO right now we ONLY show deployments / RCs if they have pods, but we may also want to show 
                        them if they are idled down to zero for example.
                   -->
-                  <div ng-if="(podsByDeployment[deployment.metadata.name] | hashSize) > 0">
-                    <div class="small muted" ng-if="deployment" style="margin-bottom: 10px;">
-                      <relative-timestamp timestamp="deployment.metadata.creationTimestamp"></relative-timestamp>
-                      <span ng-if="deployment.details && deployment.details.causes && deployment.details.causes.length > 0">
-                        <span>, triggered by 
-                          <span class="deployment-trigger" ng-repeat="cause in deployment.details.causes">
-                            <span ng-switch="cause.type">
-                              <span ng-switch-when="ImageChange">new image for {{cause.imageTrigger.repositoryName | imageName}}:{{cause.imageTrigger.tag}}</span>
-                              <span ng-switch-when="ConfigChange">deployment configuration change</span>
-                            </span>
+                  <div class="small muted" ng-if="deployment" style="margin-bottom: 10px;">
```

Comment:
Is the `TODO right now we ONLY show deployments / RCs if they have pods` above still accurate? Looks like we dropped the check of podsByDeployment being of non-zero length